config-example.properties

```diff
@@ -42,6 +42,9 @@ graph.dataaccess=RAM_STORE
 # graph but less for every request.
 # prepare.chWeighting=fastest
 #
+# You can also setup multiple weightings, by providing a coma separated list like:
+# prepare.chWeighting=fastest, shortest
```

Comment:
for the encoders we accept also spaces I think, trimming is easy and should be done before calling setCHWeighting I think. But for now I would be even okay with trimming inside the setter, not sure.